serverless/tensorflow/matterport/mask_rcnn/nuclio/main.py

```diff
@@ -0,0 +1,32 @@
+import json
+import base64
+from PIL import Image
+import io
+from model_loader import ModelLoader
+import numpy as np
+import yaml
+
+
+def init_context(context):
+    context.logger.info("Init context...  0%")
+
+    functionconfig = yaml.safe_load(open("/opt/nuclio/function.yaml"))
+    labels_spec = functionconfig['metadata']['annotations']['spec']
+    labels = {item['id']: item['name'] for item in json.loads(labels_spec)}
+
+    model_handler = ModelLoader(labels)
```

Comment:
Codacy found an issue: [No value for argument 'weights' in constructor call](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5729179)